dt-impact-tracer/src/main/java/edu/washington/cs/dt/impact/plugin2/Sample.java

```diff
@@ -0,0 +1,227 @@
+package edu.washington.cs.dt.impact.plugin2;
+
+import com.reedoei.testrunner.mavenplugin.TestPlugin;
+import com.reedoei.testrunner.mavenplugin.TestPluginPlugin;
+import org.apache.commons.io.FileUtils;
+import org.apache.maven.cli.MavenCli;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.classworlds.realm.ClassRealm;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+public class Sample extends TestPlugin {
+    // Args (Re-Declared Upon Each Use)
```

Comment:
Note, all these comments also apply to the first plugin above, I just made them here because I didn't start my review in order.